devdox/app/schemas/api_key_schema.py

```
@@ -20,3 +22,12 @@ class Config:
         title = "API Key Create Schema"
         description = "Used when creating a new API key. Includes the full hashed key along with base fields."
         from_attributes = True
+
+class APIKeyRevokeRequest:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)